nomic/dataset.py

```diff
@@ -664,14 +664,22 @@ def _download_sidecar(self, sidecar_name, overwrite: bool = False) -> List[Path]
         sidecar_suffix = "feather"
         if sidecar_name != "":
             sidecar_suffix = f"{sidecar_name}.feather"
-        for key in tqdm(self._manifest["key"].to_pylist()):
-            sidecar_path = self.tile_destination / f"{key}.{sidecar_suffix}"
-            sidecar_url = (
-                self.dataset.atlas_api_path
-                + f"/v1/project/{self.dataset.id}/index/projection/{self.id}/quadtree/{key}.{sidecar_suffix}"
-            )
-            download_feather(sidecar_url, sidecar_path, headers=self.dataset.header, overwrite=overwrite)
-            downloaded_files.append(sidecar_path)
+        with concurrent.futures.ThreadPoolExecutor(16) as ex:
```

Comment:
Let's make this more like 2-4 -- too many open will hold up multiple API workers as the bandwidth saturates.

```suggestion
        with concurrent.futures.ThreadPoolExecutor(16) as ex:
```